pkg/util/docker/docker_util.go

```diff
@@ -235,7 +235,6 @@ func (d *dockerUtil) containers(cfg *ContainerListConfig) ([]*Container, error)
 		containers, ok = cached.([]*Container)
 		if !ok {
 			log.Errorf("invalid cache format, forcing a cache miss")
-			hit = false
```

Comment:
Not sure what was the expected outcome there? (hit is not reused bellow)

Reply:
That's likely a bug. https://github.com/DataDog/datadog-agent/blob/9a735338a8c094478225860b8a371c194d029b7e/pkg/util/docker/docker_util.go#L240 This should be `if !hit`, not `else` I think.